risc0/zkvm/sdk/rust/src/host/mod.rs

```diff
@@ -257,4 +258,44 @@ mod test {
         prover.add_input_u32_slice(&[5, 5]);
         prover.run().unwrap();
     }
+
+    #[test]
+    fn sha_accel() {
+        let opts = ProverOpts::default().with_skip_seal(true);
+        let mut prover =
+            Prover::new_with_opts(&std::fs::read(SHA_ACCEL_PATH).unwrap(), SHA_ACCEL_ID, opts)
+                .unwrap();
+        prover.add_input_u32_slice(&[
+            0, // Test risc0_zkvm_guest::sha::Impl
+            0, // Compute an empty digest
+        ]);
+        prover.run().unwrap();
+    }
+
+    #[test]
+    fn insecure_sha_accel() {
+        let opts = ProverOpts::default().with_skip_seal(true);
+        let mut prover =
+            Prover::new_with_opts(&std::fs::read(SHA_ACCEL_PATH).unwrap(), SHA_ACCEL_ID, opts)
+                .unwrap();
+        prover.add_input_u32_slice(&[
+            1, // Test risc0_zkvm_guest::sha_insecure::Impl
+            0, // Compute an empty digest
+        ]);
+        prover.run().unwrap();
+    }
+
+    #[test]
+    fn sha_cycle_count() {
+        let opts = ProverOpts::default().with_skip_seal(true);
+        let mut prover =
+            Prover::new_with_opts(&std::fs::read(SHA_ACCEL_PATH).unwrap(), SHA_ACCEL_ID, opts)
+                .unwrap();
+        prover.add_input_u32_slice(&[
+            2, /* Check the cycle count on insecure sha matches what we expect from the
+                * acceleration circuit */
```

Comment:
Use `//` rather than `/*` comments

Reply:
Wild, rustfmt turns it into a /* comment when it wraps it.  Shortened. :)